components/Form/validations.js

```diff
@@ -79,6 +79,11 @@ class Validations {
     const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
     return pattern.test(value) ? '' : msg.EMAIL;
   }
+
+  static EmailBlockAccent({ value }) {
```

Comment:
I like it!
But why dont block accents in the current email validation?
Set it as default. Blocks, accent in email validation, all the time?

Reply:
Thanks. Since Quantum is open source, I think that not everyone might want to block accent. At least while I was searching, it's not ver common.  What do you think?